### ISSUE TITLE
Atom 1.7.1 keeps crashing on Mac 10.11.4

### ISSUE THREAD

**Issue by justinchristensen66** [atom/atom#11519]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

You can get this information from executing `atom --version` and `apm --version` at the command line. Also, please include the OS and what version of the OS you're running.

**Comment by justinchristensen66** [comment]
Atom crashes when I open it after a few moments. The problem is reproduced when I  click on a different window like chrome browser and then click back to the editor and Atom is frozen and then crashes.   I am on Mac OSX 10.11.4 and I am using Atom 1.7.1

**Comment by techninja** [comment]
Getting similar crashiness, 1.7.2 on both OSX and Win 10 x64. Editor will die randomly when clicking to other windows, or while typing (~10 times tonight). About the only graceful save here is that the panes and files stay put, even if I lose my buffer/edits. I do swear loudly every time though :rage1:.

**Comment by paper9oll** [comment]
Nothing happen even when i switch between Atom and Chrome when i'm doing my dev work.

Atom version - 1.7.2
macOS version - 10.11.4
Machine - rMBP (Early 2015)

**Comment by bagadim** [comment]
Same on 1.7.1 Win 7 x64. Crashes randomly on editing code, switching, using plugins.

**Comment by coderobe** [comment]
Same issue. Arch Linux x64
```
Atom    : 1.7.1
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```

**Comment by agaurav** [comment]
I am facing similar crashes too. In-fact all the file content of the active tab gets wiped as well (the entire file,along with all the saved changes)

```
Atom :   1.7.2
Chrome :   49.0.2623.112 (64-bit)
Node :  5.9.1
os :  OSX 10.10.5

```

**Comment by lee-dohm** [comment]
@justinchristensen66 Have you tried launching Atom in Safe Mode? To launch Safe Mode:

1. Completely exit all instances of Atom
1. Launch Atom with the command `atom --safe`

**Comment by lee-dohm** [comment]
@techninja @paper9oll @bagadim @coderobe @agaurav Have you tried launching Atom in Safe Mode? To launch Safe Mode:

1. Completely exit all instances of Atom
1. Launch Atom with the command `atom --safe`

For all, fair warning, "random crashing" doesn't give us much to go on. For those of you on OS X, please post crash logs in a Gist and link to them here. For other platforms, please do the same with your equivalents.

**Comment by paper9oll** [comment]
@lee-dohm i have not tried in Safe Mode but crashing does not affect me

**Comment by BrandonDusseau** [comment]
I am experiencing frequent editor crashing but thus far I have not encountered it in safe mode.

(Probably unhelpful) crash log: https://gist.github.com/BrandonDusseau/9f270a8467a8b557a79c50c0208e6f1a

Packages installed:
```
atom-fuzzy-grep
atom-minify
file-icons
highlight-selected
minimap
minimap-cursorline
minimap-find-and-replace
minimap-git-diff
minimap-highlight-selected
oceanic-dark-next [theme]
oceanic-reef-syntax [theme, enabled]
oceanic-theme [theme]
```

Atom information:
```
Windows 10 x64
Atom    : 1.7.2
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```

**Comment by lee-dohm** [comment]
@BrandonDusseau If you can't reproduce the issue in Safe Mode, then the problem is nearly guaranteed to be caused by one of the packages you have installed. Some people have found the [package-cop package](https://atom.io/packages/package-cop) to be helpful in narrowing things down.

**Comment by BrandonDusseau** [comment]
@lee-dohm Thanks for the suggestion. I'll keep watching for the problem and see if I can narrow down the culprit. Perhaps the people above have a faulting package in common with me.

**Comment by coderobe** [comment]
@lee-dohm the same crash affects me even in safe mode. afaik atom doesn't produce any crashlogs that i know of. i can not test package-cop-package because atom crashes in the first minute after opening it. i have updated atom to 
```
Atom    : 1.9.0-dev-9ff5fb2
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```
this didn't resolve the issue.

`~/.atom/nohup.out` contains `Failed to get crash dump id.`

**Comment by lee-dohm** [comment]
@coderobe Please open a new Issue.

**Comment by justinchristensen66** [comment]
Yes, I followed the outlined list before filing the issue.

**Comment by lee-dohm** [comment]
@justinchristensen66 Can you give any more information as to what environment you're running in? Currently, I cannot reproduce the issue you're describing.

* What size project are you opening?
* What size and language file are you editing?
* Is there anything else you can tell us?

**Comment by justinchristensen66** [comment]
Hi Lee, I think I solved the problem.  I installed plugin for
autocomplete-elixir and as long as that plugin is disabled the problem goes
away.  I found explanations on the github page that wasn't on the atom
plugin page about setting paths for elixir and erlang concerning Mac
OSX, you insert the whole path for elixir and only the folder path for
Erlang. I think this clears things up, if the problem continues it is for
sure the plug-in causing the problem.  Thanks for following up and sorry
for the delayed response.
